shared/organization.py

```diff
@@ -5,8 +5,16 @@
 # maximum allowed value, c.f. https://docs.aws.amazon.com/organizations/latest/APIReference/API_ListAccounts.html#API_ListAccounts_RequestSyntax
 MAX_NUM_RESULTS = 20
 
-def get_organization_accounts():
-  organizations_client = boto3.client('organizations')
+def get_organization_accounts(**arguments):
+
+  if (arguments and arguments['profile_name']):
+    session_data = {}
+    session_data["profile_name"] = arguments['profile_name']
+    session = boto3.Session(**session_data)
```

Comment:
You really like the `**` syntax, `session_data = {}` probably isn't needed here, right?

I am unsure if we do `get_organization_accounts(**vars(arguments))` anywhere else in the codebase, do you know?